### ISSUE TITLE
Is Codabar suppprted?

### ISSUE THREAD

**Issue by OBXS** [macherel/Barcode-Wallet#30]
I'm having trouble reading Codabar barcodes using the data-manager site, also I don't see the Codabar as a valid type in the dropdown menu so I still can't recreate the barcode I want even if I input the data manually. Just want to make sure if it's currently supported. Normal QR code works no problem so I don't think it's a problem with my device.

**Comment by macherel** [comment]
All barcode types in the following list are supported (even if it's not in the dropdown list) : https://github.com/metafloor/bwip-js/wiki/BWIPP-Barcode-Types

For codabar, you have to put "rationalizedCodabar" in the type field.
Note that some scanner don't display the whole value : a codabar value usually starts and ends with a "A" (it can be also a "B" "C" or a "D") so your value should look like "A12345678A".

**Comment by OBXS** [comment]
Quick update! it turns out that Google Wallet and another third party app I used only shows the Leading and Last Delimiters (the A, B, C, D) on some barcodes, randomly hiding them it seems. After trying a bunch of online readers I was able to find one that showed the full decoded data on the problematic ones and now it works no problem!

The only remaining problem now is that sometimes the barcode doesn't get read by a scanner but I think that's more of a hardware issue (AKA my watch being not big enough for the scanner)
